### ISSUE TITLE
PyPlot crashes after Julia MKL is installed

### ISSUE THREAD

**Issue by ChrisNabold** [JuliaLinearAlgebra/MKL.jl#46]
I installed PyPlot 
tested with PyPlot example -> works fine
I instatalled LinearAlgebra + MKL no error message
rebooted Julia
Tried the PyPLot example --> Julia windows suddenly closes withot error message

**Comment by aminya** [comment]
Try #35

**Comment by doorisajar** [comment]
Before #35, I was getting the following when using `PyCall` to import a package that depends on `numpy`:

```
julia> using PyCall
julia> np = pyimport("numpy")

Intel MKL FATAL ERROR: Cannot load libmkl_intel_thread.dylib.
```

After updating `MKL.jl` and rebuilding it, now I get:

```
julia> using PyCall
julia> np = pyimport("numpy")

Intel MKL ERROR: Parameter 12 was incorrect on entry to DGELSD.
init_dgelsd failed init
```

This is all on Julia 1.5, using `numpy` installed via `Conda.jl` in the root environment solely for use with `PyCall`.

**Comment by ViralBShah** [comment]
This should be fixed in Julia 1.7 which uses the LBT api. Please reopen if still an issue in Julia 1.7 or later.
